test/unit/content-src/components/Search.test.jsx

```diff
@@ -102,5 +102,33 @@ describe("<Search>", () => {
       });
       assert.calledWith(dispatch, {type: "FOCUS_SEARCH"});
     });
+    it("should hand-off search when user types", () => {
+      const dispatch = sinon.spy();
+      const wrapper = shallowWithIntl(<Search {...DEFAULT_PROPS} handoffEnabled={true} dispatch={dispatch} />);
+      wrapper.instance().onSearchHandoffKeyDown({key: "f"});
```

Comment:
Could you use [`simulate("keydown", "f")`](https://airbnb.io/enzyme/docs/api/ShallowWrapper/simulate.html) so it's closer to the real user action.

Reply:
ah, good point 👍